packages/core-components/src/components/toggle-switch/toggle-switch.scss

```diff
@@ -0,0 +1,128 @@
+@use '../../global/b2b-styles';
+
+$switch-width: 40px;
```

Comment:
Great use of local SCSS variables! If possible, use the b2b tokens for encapsulation here too instead of raw pixel values.

Reply:
done